packages/web/src/app/doc-pages/brand/color-doc/color-picker/color-picker-header/color-picker-header.component.ts

```diff
@@ -12,5 +14,12 @@ import { LocalThemeSwitchComponent } from 'src/app/shared/local-theme-switch/loc
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
 })
 export class ColorPickerHeaderComponent {
-  changeThemeEvent = output<ThemeName>();
+  private colorPickerService = inject(ColorPickerService);
+  theme = toSignal(this.colorPickerService.theme$, {
+    initialValue: 'light',
+  });
```

Comment:
Is this used anywhere?

Reply:
I just use it to set the background of the main container